### PR TITLE
feat(integrations): add support for eBay

### PR DESCRIPTION
**Pull request by cassanelligiovanni**
##Describe your changes

Added configuration for `ebay` and `ebay-sandbox` in providers.yaml
Added `ebay` to docs-v2.
Added `ebay` pages for Supported API groups in mint.json.
Added `ebay` and `ebay-sandbox`  SVG file in template-logos folder.
Consolidate Ebay docs pages

##Test

This provider has been used successfully in local development to connect to Ebay Sandbox API  and Ebay API and successfully call those endpoints after authorizing using Nango. (Have not tested sync and actions)

### CODE REVIEW

File: packages/webapp/public/images/template-logos/ebay-sandbox.svg
Comment (bodinsamuel) [inline packages/webapp/public/images/template-logos/ebay-sandbox.svg:1]: can you use a symlink instead please? ☺️

File: packages/shared/providers.yaml [hunk 1]
```diff
@@ -1990,6 +1990,44 @@ emarsys:
         expires_in_ms: 300000
     docs: https://docs.nango.dev/integrations/all/emarsys
 
+ebay:
+    display_name: eBay
+    categories:
+        - marketplace
+        - e-commerce
```
Comment (bodinsamuel) [inline packages/shared/providers.yaml:2059]: Can you add a reference to this integration in the corresponding mdx (ecommerce.mdx) and remove `marketplace` that does not exist in our system?

**Review by bodinsamuel — COMMENTED** [review]
Thanks for your contribution 🙏🏻 
Small comments but otherwise looks good

File: packages/webapp/public/images/template-logos/ebay-sandbox.svg
Comment (cassanelligiovanni) [reply packages/webapp/public/images/template-logos/ebay-sandbox.svg:1]: How can i create a symlink? do you have examples of past contributions? Cause i was looking at Brex integration and they have 2 identical logos

Comment (hassan254-prog) [reply packages/webapp/public/images/template-logos/ebay-sandbox.svg:1]: You can create a symbolic link with the following command within the `templates-logos` folder
`ln -s ebay.svg ebay-sandbox.svg`

File: packages/shared/providers.yaml (hunk 1 above)
Comment (cassanelligiovanni) [reply packages/shared/providers.yaml:2059]: Hi, I committed, not a very expert with forks. Can you see updates? Do i have to do something else?

**Review by bodinsamuel — APPROVED** [review]
Perfect, merging that and deploying during the day 👍🏻

**Pull request merged**